### PR TITLE
Fix wrong texts in titles when cloning templates

### LINKED ISSUE #63503: Duplicating a notification template shows "Edit notification template" instead of "Create notification template"
**What happened**:

Clicking Copy template on a notification template shows "Edit notification template" instead of "Create notification template".

**What you expected to happen**:

The UI should show "Create notification template". This would be consistent with copying an alert rule which says "Add rule" instead of "Edit rule".

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version:
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

### PR DESCRIPTION
**Pull request by soniaAguilarPeiron**
**What is this feature?**

This PR fixes wrong text when cloning a template. The UI should show "Create notification template" instead of "Edit notification template".

**Why do we need this feature?**

UI should show "Create notification template" in the title when cloning a rule.

**Who is this feature for?**

All users.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/63503

**Special notes for your reviewer**:

https://user-images.githubusercontent.com/33540275/222134320-b2d7d271-ce18-480b-808a-512934299299.mp4

### CODE REVIEW

File: public/app/features/alerting/unified/Receivers.tsx [hunk 1]
```diff
@@ -181,8 +182,14 @@ const Receivers = () => {
   );
 };
 
-function getPageNavigationModel(type: PageType | undefined, id: string | undefined) {
+function getPageNavigationModel(type: PageType | undefined, id: string | undefined, isDuplicatingTemplates: boolean) {
   let pageNav: NavModelItem | undefined;
+  if (isDuplicatingTemplates) {
+    return {
+      text: `New template`,
+      subTitle: `Create a new temaplate for your notifications`,
```
Comment (gillesdemey) [inline public/app/features/alerting/unified/Receivers.tsx:190]:
```suggestion
      subTitle: `Create a new template for your notifications`,
```

File: public/app/features/alerting/unified/components/receivers/TemplateForm.tsx [hunk 2]
```diff
@@ -37,6 +39,7 @@ interface Props {
   alertManagerSourceName: string;
   provenance?: string;
 }
+export const isDuplicating = (location_: Location) => location_.pathname.endsWith('/duplicate');
```
Comment (gillesdemey) [inline public/app/features/alerting/unified/components/receivers/TemplateForm.tsx:42]: any particular reason why we're prefixing here with `_`?

**Review by gillesdemey — CHANGES_REQUESTED** [review]
Needs a typo to be fixed :)

File: public/app/features/alerting/unified/Receivers.tsx (hunk 1 above)
Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/Receivers.tsx:190]: ohhhhh!! got catch!!

File: public/app/features/alerting/unified/components/receivers/TemplateForm.tsx (hunk 2 above)
Comment (soniaAguilarPeiron) [reply public/app/features/alerting/unified/components/receivers/TemplateForm.tsx:42]: you're right, we can remove the prefix (at the beginning I had the method inside the component where we had another location const defined, that's why I added the prefix. But not it's not necessary)

**Review by gillesdemey — APPROVED** [review]

**Pull request merged**